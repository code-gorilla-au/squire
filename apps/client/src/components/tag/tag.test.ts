import { render, screen } from "@testing-library/svelte";
import { describe, expect, it } from "vitest";

import Tag from "./tag.test.svelte";

describe("title component", () => {
	it("should render the tag", () => {
		render(Tag);

		expect(screen.getByText("Test Tag")).toBeInTheDocument();
	});
	it("should match snapshot", () => {
		const { container } = render(Tag);
		expect(container.innerHTML).toMatchSnapshot();
	});
});
