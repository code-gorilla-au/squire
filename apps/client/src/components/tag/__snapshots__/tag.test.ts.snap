// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`title component > should match snapshot 1`] = `"<span class="text-xs bg-muted p-1 text-muted-foreground rounded-lg">Test Tag<!----></span>"`;
